File: src/test/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/KafkaUtsendingHistorikkData.java
package no.nav.arbeidsgiver.sykefravarsstatistikk.api;

import java.time.LocalDateTime;

public class KafkaUtsendingHistorikkData {

  public String orgnr;
  public String key;
  public String value;
  public LocalDateTime opprettet;

  public KafkaUtsendingHistorikkData(
      String orgnr,
      String key,
      String value,
      LocalDateTime opprettet
  ) {
    this.orgnr = orgnr;
    this.key = key;
    this.value = value;
    this.opprettet = opprettet;
  }
}

File: src/test/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/KafkaUtsendingHistorikkData.kt
package no.nav.arbeidsgiver.sykefravarsstatistikk.api

import java.time.LocalDateTime

class KafkaUtsendingHistorikkData(
  @JvmField var orgnr: String, @JvmField var key: String, @JvmField var value: String, @JvmField var opprettet: LocalDateTime
)
